tests/by-util/test_install.rs

```diff
@@ -674,6 +674,34 @@ fn test_install_and_strip_with_program() {
     assert!(!stdout.contains(STRIP_SOURCE_FILE_SYMBOL));
 }
 
+#[cfg(all(unix, feature = "chmod"))]
+#[test]
+// FixME: Freebsd fails on 'No such file or directory'
+#[cfg(not(any(windows, target_os = "freebsd")))]
```

Comment:
The `windows` seems unnecessary as we already have `all(unix, ...)` on line 677:
```suggestion
#[cfg(not(target_os = "freebsd"))]
```